uiServer/ReadMe.txt

```diff
@@ -0,0 +1 @@
+Hello World
```

Comment:
@flynnmichaelj Do we want all the README edits in the top README.md file, Mike?

Reply:
Yes, i think all documention should be in the top level README.md file, including usage of UI mode.